Assets/Scripts/Actors/ActorComponents/PlayerActorInventory.cs

```diff
@@ -116,7 +120,21 @@ void SpawnBuddy()
 		BuddyStats newBuddy = ( Instantiate( buddyItemData.buddyPrefab,
 		                                     transform.position + transform.forward * 3.0f,
 		                                     Quaternion.identity ) as GameObject ).GetComponent<BuddyStats>();
```

Comment:
The formatting here is technically fine, though I'd move the `Instantiate()` call to a new line to keep it from getting overly long.